node/src/components/consensus/era_supervisor.rs

```diff
@@ -111,6 +110,7 @@ where
         let (root, config) = config.into_parts();
         let secret_signing_key = Rc::new(config.secret_key_path.load(root)?);
         let public_signing_key = PublicKey::from(secret_signing_key.as_ref());
+        let bonded_eras: u64 = DEFAULT_UNBONDING_DELAY - chainspec.genesis.auction_delay;
```

Comment:
Ah, I guess that answers my other comment: The unbonding delay is not yet in the chainspec, is it? And the locking period applies only to genesis validators?

Reply:
Yes, the story only moved some constants out into the chainspec.